examples/src/test/kotlin/SortTest.kt

```diff
@@ -121,49 +122,49 @@ internal class SortTest {
     fun descendingSortTest() = runBlocking {
         // :snippet-start: descending-sort
         val resultsFlow = collection.find()
-            .sort(Sorts.descending("_id"))
+            .sort(Sorts.descending(Order::orderTotal.name))
 
         resultsFlow.collect { println(it) }
         // :snippet-end:
         // Junit test for the above code
         val expected = listOf(
-            FoodOrder(6, "c", "maple donut"),
-            FoodOrder(5, "a", "milk and cookies"),
-            FoodOrder(4, "b", "coffee with sugar"),
-            FoodOrder(3, "a", "maple syrup"),
-            FoodOrder(2, "a", "donuts and coffee"),
-            FoodOrder(1, "c", "coffee with milk")
-            )
+            Order(2, "2022-01-11", 83.87, "two medium vanilla birthday cakes"),
+            Order(5, "2022-01-23", 60.31, "one large vanilla and chocolate cake"),
+            Order(4, "2022-01-15", 43.62, "2 chicken lunches and a diet coke"),
+            Order(3, "2022-01-11",  19.49, "1 dozen vanilla cupcakes"),
+            Order(1, "2022-01-03", 17.86, "1/2 lb cream cheese and 1 dozen bagels"),
+            Order(6, "2022-01-23", 10.99, "1 bagel, 1 orange juice, 1 muffin")
+        )
         assertEquals(expected, resultsFlow.toList())
     }
 
     @Test
     fun handleTiesTest() = runBlocking {
         // :snippet-start: handle-ties-1
-        collection.find().sort(Sorts.ascending(FoodOrder::letter.name))
+        collection.find().sort(Sorts.ascending(Order::date.name))
```

Comment:
Question just for my own learning: What is the `.name` referring to here (and in other instances)? From what I can see the date field is just a string

Reply:
From the Kotlin docs on property references:
> To access properties as first-class objects in Kotlin, use the :: operator:
> ...
> You can read its value using get() or retrieve the property name using the name property

Reply:
From what I have gathered, this is a more idiomatic way of referencing the field names instead of using a string as so:
```kotlin
collection.find().sort(Sorts.ascending("date"))
```

Reply:
Thanks!